config/options.yml

```diff
@@ -48,7 +48,7 @@ rpm_repository:
         :python-unittest2: !ruby/regexp /.+-1\.1\.0.+/
         :python-vspk: !ruby/regexp /.+-5\.3\.2.+/
         :qpid-proton: !ruby/regexp /.+-0\.30\.0.+/
-        :repmgr10: !ruby/regexp /.+-4\.0\.6.+/
+        :repmgr13: !ruby/regexp /.+-5\.2\.1.+/
```

Comment:
I believe we'll need the repmgr rpm pushed to digital ocean in order to merge this.

Reply:
I don't think we need to keep the prior repmgr10 directory or this entry as it's using for building rpms for repmgr10.  I think we can live with rpms already on digital ocean but no longer provide a way to build repmgr10 going forward.  I think this is ready to go but let me know if you think otherwise.

Reply:
@bdunne let me know if you think anything here or in the PR should be altered to get merged.  I think my assumptions are correct regarding keeping existing built repmgr10 rpms on digital ocean but prevent building them going forward as we move to repmgr13.